srcpkgs/rubyripper/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'rubyripper'
+pkgname=rubyripper
+version=0.7.0rc2
```

Comment:
This is a Release Candidate, isn't 0.6.2 enough ?

Reply:
According to https://github.com/bleskodev/rubyripper#historical-note
"This is an unofficial fork of the original Rubyripper from [Google code](https://code.google.com/archive/p/rubyripper/). The code was abandoned in 2014 by the original author. I was (and still am) a user, and the latest released version (0.6.2) stopped working for me as I followed the OS upgrades."